modules/nf-core/custom/dumpsoftwareversions/environment.yml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::multiqc=1.15
```

Comment:
Not 1.17?

Reply:
https://github.com/nf-core/modules/issues/4290